media/css/firefox/new/all-test-variations-pt2.less

```diff
@@ -0,0 +1,250 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+// Common
+#test-primary-ctas {
+    ul {
+        list-style: none;
+    }
+
+    li {
+        display: inline-block;
+        margin: 0;
+    }
+}
+
+#test-mobile-downloads {
+    text-align: center;
+
+    li:first-child {
+        margin-right: 10px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        li {
+            display: list-item;
+
+            &:first-child {
+                margin-right: 0;
+            }
+        }
+    }
+}
+
+#test-direct-downloads {
+    padding: 10px 30px 0;
+    text-align: left;
+
+    li {
+        .border-box();
+        margin-bottom: 10px;
+        width: 50%;
```

Comment:
I think these may also need to float left instead of being inline-block. In smaller viewports "Windows (XP/Vista)" wraps to two lines and "OSX" is misaligned. I think floating the list items lets the text align top but there may be another fix too. 

![fxnew-v2-listalign](https://cloud.githubusercontent.com/assets/205591/21574102/061cb4c0-cea2-11e6-8799-6d0dd599fb6e.png)